cmd/immuadmin/command/commandline.go

```diff
@@ -51,7 +51,7 @@ type commandline struct {
 	newImmuClient  func(*client.Options) (client.ImmuClient, error)
 	passwordReader c.PasswordReader
 	context        context.Context
-	hds            client.HomedirService
+	ts             client.Token_service
```

Comment:
Token_service might be renamed to TokenService, and database parameterization may not be required